### DISCUSSION TITLE
Can't run outline service

### DISCUSSION THREAD

**Discussion by setimets** [outline/outline#3280] [Self Hosting] [answered]
I try to install Outline into my server as a self-hosting.

**Screenshots**

but the below logs are Outline service error logs from the docker-compose.

![image](https://user-images.githubusercontent.com/5173341/159841843-ec345b4f-42b7-46be-8721-4da7268f4bb5.png)

**Outline (please complete the following information):**
- Install: self hosted
- Version:v 0.61.0

What am I missing to install?

**Comment by 7olstoy** [comment, marked as answer]
You need to run 
```
docker-compose run --rm outline yarn sequelize db:create --env=production-ssl-disabled
docker-compose run --rm outline yarn sequelize db:migrate --env=production-ssl-disabled
```
before main `docker-compose up -d`

**Reply by almereyda** [reply]
Yes.

More details about the Docker installation procedure can be found at https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/docker-7pfeLP5a8t

To find this link, you can first

- read the readme's *Installation* section at https://github.com/outline/outline/blob/main/README.md#installation
- go to the *Hosting outline* document linked from there at https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/
- check out the *Installation methods* as presented on https://app.getoutline.com/share/770a97da-13e5-401e-9f8a-37949c19f97e/doc/installation-methods-pSvgz9j0QC
- to finally find the Docker link there.

Indeed this could be a little more obvious and less hidden, as these instructions seem to be disregarded in multiple occasions already, and Docker Compose seems to be the major installation method used by people.

**Reply by tommoor** [reply]
> Indeed this could be a little more obvious and less hidden, as these instructions seem to be disregarded in multiple occasions

You're learning about the joys of OSS 😆 , thank you for your help.

**Reply by almereyda** [reply]
It's pure love, don't you worry. 💞

**Reply by setimets** [reply]
Works fine ~ Thanks a lot.

**Comment by almereyda** [comment]
@setimets If any of the above ansers your request, don't hesitate to mark this conversation as answered.
